frontend/packages/operator-lifecycle-manager/src/components/operand.tsx

```diff
@@ -60,7 +60,11 @@ const csvName = () =>
 const getActions = (selectedObj: any) => {
   const actions = plugins.registry
     .getClusterServiceVersionActions()
-    .filter((action) => action.properties.kind === selectedObj.kind);
+    .filter(
+      (action) =>
+        action.properties.kind === selectedObj.kind &&
+        _.startsWith(selectedObj.apiVersion, action.properties.apiVersion),
```

Comment:
Don't use `startsWith` since it could incorrectly match a substring. Use `groupVersionFor`

```suggestion
        groupVersionFor(selectedObj) === action.properties.apiGroup
```

Reply:
ack